.changeset/swift-lies-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query-builder': patch
+---
+
+Fix a regression introduced in ([#1543](https://github.com/finos/legend-studio/pull/1543)).
```

Comment:
Can we briefly mention what this regression is about so people can figure it out from the changelog rather than having to go to the link?